app/service-catalog/integrations/gitlab.md

```diff
@@ -0,0 +1,58 @@
+---
+title: GitLab
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+works_on:
+    - konnect
+description: The GitLab integration allows you to associate your Service Catalog service to one or more [GitLab projects](https://docs.gitlab.com/ee/user/get_started/get_started_projects.html). 
+
+related_resources:
+  - text: "Service Catalog"
+    url: /service-catalog/
+discovery_support: true
+bindable_entities: "Projects"
+---
+
```

Comment:
```suggestion
The GitLab integration allows you to associate your Service Catalog service to one or more [GitLab projects](https://docs.gitlab.com/ee/user/get_started/get_started_projects.html).

```
We don't display the `page.description` on the page, so you have to add it here.